configs/efficientdet/retinanet_effd0_bifpn_1x_coco.py

```diff
@@ -109,8 +109,8 @@
         ])
 ]
 data = dict(
-    samples_per_gpu=12,
-    workers_per_gpu=6,
+    samples_per_gpu=20,
```

Comment:
Per GPU batch size has increased (12 -> 20), but memory consumption has dropped (22.0 -> 14.2 GB). This is counterintuitive. Please double check the numbers.

Reply:
Original batch size was 32 (where 22.0 GB was reported), changed to 12 when testing on local machine (which was not as powerful as machine used for training). Forgot to change back when committing.